html-css/portfolio.css

```diff
@@ -0,0 +1,136 @@
+
+body {
+    font-family: Arial, sans-serif;
+    color: #333;
+    background-color: #fafafa;
+    line-height: 1.6;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+nav ul {
+    display: flex;
+    justify-content: center;
+    background-color: beige;
+    padding: 10px;
+}
+
+nav ul li {
+    margin: 0 15px;
+}
+
+nav ul li a {
+    color: black;
+    text-decoration: none;
+    font-weight: bold;}
+header {
+    text-align: center;
+    padding: 60px 20px;
+    background-color: cadetblue;
+    color: #fff;
+    width: 100%;
+    animation: fadeIn 2s ease-in;
+}
+header img {
+    max-width: 150px;
+}
+header h1 {
+    font-size: 1.8rem;
+    margin: 10px;
+}
+
+header p {
+    font-size: 1rem;
+    font-weight: lighter;
+}
+
+.about-me {
+    padding: 40px;
+    max-width: 800px;
+    text-align: center;
+    width: 100%;
+}
+
+.about-me h2 {
+    font-size: 1.8rem;
+    margin-bottom: 20px;
+    color: #333;
+}
+.about-me p {
+    font-size: 1rem;
+    margin-bottom: -30px;
+    color: #333;
+}
+.skills {
+    font-size: 1.5rem;
+    margin: 20px;
+    color: #333;
+    text-align: center;
+}
+.skills p {
+    font-size: 1rem;
+}
+.skills-container {
+    display: flex;
+    justify-content: center;
+    gap: 10px;
+}
+
+.skill {
+    background-color: teal;
+    color: #fff;
+    padding: 10px 20px;
+    border-radius: 5px;
+    font-weight: bold;
+}
+
+.Projects {
```

Comment:
Update .Projects to .projects like I stated earlier to maintain consistent naming conventions and should be a good idea to add padding for layout.